packages/relay/tests/lib/subscriptionController.spec.ts

```diff
@@ -141,10 +117,10 @@ describe("subscriptionController", async function() {
 
 
     it('notifySubscribers should notify multiple subscribers with data', async function () {
-        const wsConnection1 = new MockWsConnection("2");
+        const wsConnection1 = new MockWsConnection("12");
```

Comment:
Q: what was the significance of the 12 and 13 change?
Something to comment or turn into a self describing variable?

Reply:
This change broke this other UT because we were having 2 connections with the same ID and the same exact filters, so naturally the subscriptionController recognized it as repeated subscription from same connection 😅.
With changing just 1 of 2 the issue was resolved, since the tests are independent from one another, but I like to keep them with numbers different than the others just to play safe.

Reply:
Ah the number is a subscription Id.
Okay we should make that clear by setting the value to a const with an appropriate name in the tests so it's clear.

Reply:
actually is the connection Id. but yeah, you are right... it would be nice.